src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.sscs.service.hmc.topic;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.exception.HmcEventProcessingException;
+import uk.gov.hmcts.reform.sscs.model.hmcmessage.HmcMessage;
+
+@Service
+public class ProcessHmcMessageService {
+
+    public void processEventMessage(HmcMessage hmcMessage) throws HmcEventProcessingException {
```

Comment:
Here right now due to wanting to write tests. Full version will come in https://github.com/hmcts/sscs-hearings-api/pull/163